src/hotspot/share/gc/shenandoah/shenandoahHeap.hpp

```diff
@@ -731,6 +744,12 @@ class ShenandoahHeap : public CollectedHeap {
 
   void set_young_lab_region_flags();
 
+  inline void set_old_region_surplus(size_t surplus) { _old_regions_surplus = surplus; };
```

Comment:
I'd avoid the unnecessary inconsistency between `set_old_region_surplus` and `_old_regions_surplus`. I prefer both to use the singular form of `region`.